draft-ietf-quic-recovery.md

```diff
@@ -214,6 +214,13 @@ Most TCP mechanisms implicitly attempt to infer transmission ordering based on
 TCP sequence numbers - a non-trivial task, especially when TCP timestamps are
 not available.
 
+### Clearer end to a loss epoch
+
+QUIC may reduce cwnd several times on successive losses whereas TCP will keep
```

Comment:
s/cwnd/congestion window

Also, this doesn't match the title of this section. I would not talk about the cwnd reduction here, instead focus on what a loss epoch is -- that no congestion window action should occur more than once per roundtrip time, since a controller only sees the response to any action it takes about a roundtrip time later. TCP and QUIC use packet numbers to figure out when this roundtrip ends, but QUIC does it more accurately than TCP, especially when multiple losses occur within the same roundtrip time.

Reply:
Thanks, PTAL.